docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -1196,9 +1162,36 @@ The following table lists all the search category attributes that the server can
 | `MC_ABILITY_CLIENT_CUSTOM`       | Ability for custom event                 |
 | `MC_ABILITY_SEARCH`              | Ability for search                       |
 
-> **Note**
->
-> These attributes support Tizen 5.5 and Higher for Mobile and Wearable.
+> [!NOTE]
+> These attributes support Tizen 5.5 and Higher.
+
+## Media Controller Result Code Attributes
+
+The following table lists all the result code attributes that the client and server can receive:
+
+**Table: Media controller result code attributes**
+
+| Attribute                               | Description                                             |
+|-----------------------------------------|---------------------------------------------------------|
+| **Result code**                         |                                                         |
+| `MC_RESULT_CODE_SUCCESS`                | The command or the event has completed successfully |
+| `MC_RESULT_CODE_ALREADY_DONE`           | The command or the event had executed previously |
```

Comment:
@jiyong2 
Slight change to maintain consistency.
Kindly change **had** to to **has**

| `MC_RESULT_CODE_ALREADY_DONE`           | The command or the event has executed previously |

You have just missed this.

Reply:
Thanks, I missed it. I updated it to "has".

Reply:
@jiyong2 thanks